HISTORY.md

```diff
@@ -5,6 +5,7 @@
 
 ### New Features
 * Add avoid_flush_during_shutdown option, which speeds up DB shutdown by not flushing unpersisted data (i.e. with disableWAL = true). Unpersisted data will be lost. The options is dynamically changeable.
+* Add DB::SetDBOptions() to dynamically change `delayed_write_rate` option.
```

Comment:
Move it to "Public API Changes" section and say something like "Support dynamically change delayed_write_rate via SetDBOptions()".